notesapi/v1/tests/test_views.py

```diff
@@ -126,6 +139,29 @@ def test_create_note(self, mock_datetime):
 
         self.assertEqual(response.data['user'], TEST_USER)
 
+    @patch('django.conf.settings.ES_DISABLED', True)
```

Comment:
i'm glad this support is being tested...under what scenarios would we expect ES to be disabled by configuration?

In the event of ES service interruptions or cluster health issues in production, should we be able to fall back gracefully to this state, so that end users can still work?